sdk/keyvault/keyvault-certificates/review/keyvault-certificates.review.api.md

```diff
@@ -0,0 +1,286 @@
+## API Report File for "@azure/keyvault-certificates"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface CertificateIssuer {
```

Comment:
Is an empty object a valid CertificateIssuer?

Reply:
I think this is CertificateIssuerProperties?

Reply:
It does feel really weird that everything in almost every object returned is optional, so you have to do endless null checks